src/test/java/com/heatheryou/httpserver/ServerTest.java

```diff
@@ -64,7 +65,6 @@ public void serverAnswersSimpleOptions() {
                 "Allow: OPTIONS,HEAD,GET",
                 "Content-Length: 0",
                 "",
-                "",
                 ""
         });
 
```

Comment:
Generally speaking, a change of this size should also prompt some amount of commensurate test coverage. If you have to change your tests for changed behavior, but no tests fail and there are now new ones, you're missing coverage.